### PR TITLE
made changes so that root can powercylce any nodes for any reservation

### PR DESCRIPTION
**Pull request by aherna**

### CODE REVIEW

File: src/igor/power.go [hunk 1]
```diff
@@ -126,7 +126,7 @@ func runPower(cmd *Command, args []string) {
 
 			resID := currentSched.Nodes[index-1]
 			for _, r := range Reservations {
-				if r.ID == resID && r.Owner == user.Username {
+				if r.ID == resID && r.Owner == user.Username || user.Username == "root" {
```
Comment (jcrussell) [inline src/igor/power.go]: This should be `r.ID == resID && (r.Owner == user.Username || user.Username == "root")`, right?

https://play.golang.org/p/RTaMWHEb6IP

Comment (aherna) [reply src/igor/power.go]: wouldnt it be equivalent because of commutative property going left to right?

T && F || T = T && (F||T)

Comment (aherna) [reply src/igor/power.go]: the first check is a fast fail and the other two would both need to fail to through the error which is what we want

Comment (aherna) [reply src/igor/power.go]: wrong thread^

**Pull request merged**